### PR TITLE
Include blockly as a normal npm module

### PR DESCRIPTION
**Pull request by balderdash**
Instead of just including the built blockly files directly on the page and expecting it to be defined globally from apps code, we can now `import Blockly from '@code-dot-org/blockly'`. (At the moment it still gets its own entry point which just sticks it on the `window` object, but this is a step towards cleaning up its usage as a global variable.)

### CODE REVIEW

**Comment by Hamms** [comment]
This is Amazing

File: apps/src/sites/studio/pages/blockly.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import Blockly from '@code-dot-org/blockly';
+window.Blockly = Blockly;
```
Comment (Hamms) [inline apps/src/sites/studio/pages/blockly.js:2]: am I understanding correctly that this is here to make Blockly still available on the window object like it used to be, and that the next steps are to add `import Blockly` to all the code that has Blockly as a dependency, then remove this placeholder?

Comment (balderdash) [reply apps/src/sites/studio/pages/blockly.js:2]: That's the idea, yup.

**Comment by balderdash** [comment]
Just verified that Spanish is working locally.

The string loading process will eventually have to change, since the built `blockly_locale.js` files expect a global 'Blockly.Msg' object to be available. It works for now thanks to the `window.Blockly = Blockly;`bit.

**Pull request merged**